apps/reaper/lib/reaper/auth_retriever.ex

```diff
@@ -4,6 +4,26 @@ defmodule Reaper.AuthRetriever do
   """
   alias Reaper.Collections.Extractions
   alias Reaper.Cache.AuthCache
+  alias Reaper.UrlBuilder
+
+  def authorize(dataset_id, url, body, encode_method, headers, cache_ttl) do
+    cache_ttl = cache_ttl || 10_000
```

Comment:
does the `\\` default not work for this? or is this some weird `nil` value thing?

Reply:
Defaults will only work if you dont pass in nil.